CONTRIBUTING.md

```diff
@@ -40,16 +40,32 @@ To create an install from the source tree you can run either of the following:
 
     ./gradlew install -Pgradle_installPath=/usr/local/gradle-source-build
 
-This will create a minimal installation; just what's needed to run Gradle (i.e. no docs).
+This will create a minimal installation; just what's needed to run Gradle (i.e. no sources nor docs).
 
 You can then build a Gradle based project with this installation:
 
     /usr/local/gradle-source-build/bin/gradle «some task»
 
-To create a full installation (includes docs):
+To create a full installation (includes sources and docs):
 
     ./gradlew installAll -Pgradle_installPath=/usr/local/gradle-source-build
 
+### Building a distribution from source
+
+To create a Gradle distribution from the source tree you can run either of the following:
+
+    ./gradlew :distributionsFull:binDistributionZip
+
+This will create a minimal distribution in `subprojects/distributions-full/build/distributions/gradle-<version>-bin.zip`, just what's needed to run Gradle (i.e. no sources nor docs).
+
+You can then use it as a Gradle Wrapper local distribution in a Gradle based project by using a `file:/` URL pointing to the built distribution:
+
+    ./gradle wrapper --gradle-distribution-url=file:/path/to/gradle-<version>-bin.zip
+
+To create a full distribution (includes sources and docs):
+
+    ./gradlew :distributionsFull:allDistributionZip
```

Comment:
Since you added a section already, I would go all the way and add the line to configure the wrapper to use the full distribution.

Reply:
👍 done